### PR TITLE
chore: adjusting e2e test cases with a3p:latest

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR:
- Replaces `image: ghcr.io/agoric/agoric-3-proposals:main` with `image: ghcr.io/agoric/agoric-3-proposals:latest`
- Modifies the test cases for the use with `image: ghcr.io/agoric/agoric-3-proposals:latest`

### CODE REVIEW

File: test/e2e/specs/test.spec.js [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
   context('Test commands', () => {
-    let collateralizationRatio;
-    it(`should setup Keplr account and connect with Agoric Chain`, () => {
-      cy.origin('https://wallet.agoric.app/', () => {
-        cy.visit('/');
-      });
-      cy.acceptAccess().then(taskCompleted => {
+    it('should set up wallet', () => {
+      cy.setupWallet({
+        secretWords:
```
[the remark this answers is not in the document]
Comment (rabi-siddique) [reply test/e2e/specs/test.spec.js]: We're not using `@agoric/synthetic-chain` in dapp-inter. Perhaps we should source it from `config.ts` like this:
https://github.com/Agoric/dapp-inter/compare/2f0e475855c1964c9aed27add8efb84ebb5e944a..4b1abf088164dce0fa06d0edb41486ff590cc2a9

[the remark this answers is not in the document]
Comment (turadg) [reply test/e2e/specs/test.spec.js]: That's an improvement. However, it's called a "mnemonic" and the name suggests there's only one. Consider leaving room for more accounts,
```
```suggestion
export const mnemonics = {
  user1: 'tackle hen gap lady bike explain erode midnight marriage wide upset culture model select dial trial swim wood step scan intact what card symptom';
  }  
```

though I'd be fine punting this to use exports from the synthetic-chain lib. If you opt for that, please leave an `UNTIL https://github.com/Agoric/agoric-3-proposals/issues/154` comment.

File: test/e2e/specs/test.spec.js [hunk 2]
```diff
@@ -1,51 +1,58 @@
...
 
-    it('should connect with the wallet', () => {
+    it('should create a new vault and approve the transaction successfully', () => {
       cy.visit('/');
+      cy.contains('button', /ATOM/).click();
 
-      cy.contains('Connect Wallet').click();
-      cy.get('label.cursor-pointer input[type="checkbox"]').check();
-      cy.contains('Proceed').click();
+      cy.contains('.input-label', 'ATOM to lock up *')
```
[the remark this answers is not in the document]
Comment (rabi-siddique) [reply test/e2e/specs/test.spec.js]: Updated

**Comment by rabi-siddique** [comment]
UNTIL https://github.com/Agoric/agoric-3-proposals/issues/154

**Review by turadg — APPROVED** [review]

**Review by samsiegart — APPROVED** [review]

**Pull request merged**